hpx/lcos/detail/future_data.hpp

```diff
@@ -194,75 +196,6 @@ namespace detail
         std::size_t& count_;
```

Comment:
I think the `handle_continuation_recursion_count` is not needed anymore.

Reply:
I take that back, sorry.

Reply:
But it can be moved to the source file now (out of the header) ...

Reply:
good idea.